3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,81 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+const indexOfnameToRemove = names.indexOf(nameToRemove);
+names.splice(indexOfnameToRemove, 1);
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+/**When will we be there?? */
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function travelTimecalculation(travelInformation) {
```

Comment:
Consider a more consistent **camelCase** naming convention for the function name, like `calculateTravelTime`.